examples/trading/api/helpers/cancel-escrow.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { TransactionBlock } from '@mysten/sui.js/transactions';
+
+import { CONFIG } from '../config';
+import { getActiveAddress, signAndExecute } from '../sui-utils';
+
+/// Demo PTB to cancel an escrow.
```

Comment:
Out of interest, is `///` a valid doc comment designator for TS/JS?

Reply:
Agh, no. Brain lag 😆 Let me correct them all, and thanks for flagging.

The (recommended?) way for TS is https://tsdoc.org/

```typescript
/**
 * A comment
 * @param test ...
 */
```